src/main/g8/src/test/scala/$package$/AppTest.scala

```diff
@@ -5,6 +5,8 @@ import org.scalajs.dom.document
 
 import org.scalatest.FunSuite
 
+import $package$.App
```

Comment:
This shouldn't be necessary, since we're already in `$package$`.

Reply:
Hi @shadaj. I know, I thought it was weird that I needed it but I couldn't get it to compile without it. Maybe something else is going on?

Reply:
Never mind, I just removed it compiled locally. So I'll push the change.